src/containers/OrderBook/OrderBookTableRow.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { Decimal } from '../../components/Decimal';
+
+const OrderBookTableRowComponent = props => {
+    const { type, fixed, prevValue, price, total } = props;
+
+    if (type === 'price') {
+        return <span><Decimal fixed={fixed} prevValue={prevValue}>{price}</Decimal></span>;
```

Comment:
why do you use <span> tag here?

Reply:
I copied this part of a code and moved it to a separate component, didn't change anything with a structure because there can be some styles depending on this tag